hadoop-hdds/common/src/test/java/org/apache/hadoop/ozone/common/TestChecksumByteBuffer.java

```diff
@@ -59,11 +59,10 @@ void testCorrectness() {
 
       checkBytes("hello world!".getBytes(StandardCharsets.UTF_8));
 
-      final Random random = new Random();
       final byte[] bytes = new byte[1 << 10];
       for (int i = 0; i < 1000; i++) {
-        random.nextBytes(bytes);
-        checkBytes(bytes, random.nextInt(bytes.length));
+        RandomUtils.nextBytes(bytes.length);
+        checkBytes(bytes, RandomUtils.nextInt(0, bytes.length));
```

Comment:
`nextBytes` returns the byte array, which is ignored here.  So `bytes` is the same in each iteration instead of having random content.

Since this results in a new array in each iteration, we can remove the original `bytes` array.

```java
      final int len = 1 << 10;
      for (int i = 0; i < 1000; i++) {
        checkBytes(RandomUtils.nextBytes(len), RandomUtils.nextInt(0, len));
      }
```

Reply:
Thanks for the suggestion, it is corrected.